test_settings.py

```diff
@@ -22,12 +23,13 @@ def root(*args):
 COMPLETION_AGGREGATOR_BLOCK_TYPES = {'course', 'chapter'}
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'default.db',
-        'USER': '',
-        'PASSWORD': '',
-        'HOST': '',
-        'PORT': '',
+        'ATOMIC_REQUESTS': True,
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'completion_aggregator_test',
+        'HOST': environ.get('MYSQL_HOST', '127.0.0.1'),
+        'PORT': int(environ.get('MYSQL_PORT', 3306)),
+        'USER': environ.get('MYSQL_USER', 'root'),
+        'PASSWORD': environ.get('MYSQL_PASSWORD', ''),
```

Comment:
Nit: Should these be prefixed with e.g., `EDXAGG_`, so they don't conflict with other possible uses?